### PR TITLE
DE Server Blocklist docs

### PR DESCRIPTION
**Pull request by tobiasd**
German docs alongside #3392

### CODE REVIEW

File: doc/de/Settings.md [hunk 1]
```diff
@@ -238,6 +238,12 @@ Aber keine Panik!
 Friendica wird die Beiträge nicht für alle Zeiten in der Warteschlange behalten.
 Nach einiger Zeit werden Knoten als inaktiv identifiziert und Nachrichten an Nutzer dieser Knoten aus der Warteschlange gelöscht.
 
+## Server Blockliste
+
+Auf dieser Seite des Admin Panels können Administratoren einer Friendica Instanz die komplette Kommunikation (eingehdnd und ausgehend) mit bestimmten Domains unterbinden.
```
Comment (annando) [inline doc/de/Settings.md]: Der Rechtschreib-Nerd in mir merkt an, dass es "Friendica-Instanz" heißen muss ;-)

**Comment by tobiasd** [comment]
On Thu, 27 Apr 2017 01:46:37 -0700
Michael Vogel <notifications@github.com> wrote:

> Der Rechtschreib-Nerd in mir merkt an, dass es "Friendica-Instanz"
> heißen muss ;-)

Ok :-)

File: doc/de/Settings.md [hunk 2]
```diff
@@ -238,6 +238,12 @@ Aber keine Panik!
 Friendica wird die Beiträge nicht für alle Zeiten in der Warteschlange behalten.
 Nach einiger Zeit werden Knoten als inaktiv identifiziert und Nachrichten an Nutzer dieser Knoten aus der Warteschlange gelöscht.
 
+## Server Blockliste
+
+Auf dieser Seite des Admin Panels können Administratoren einer Friendica-Instanz die komplette Kommunikation (eingehdnd und ausgehend) mit bestimmten Domains unterbinden.
```
Comment (Hypolite) [inline doc/de/Settings.md]: *eingehend

Comment (annando) [inline doc/de/Settings.md]: @Hypolite understanding German is your hidden talent? ;-)

Comment (Hypolite) [inline doc/de/Settings.md]: I did 7 years of german at school, and while I wouldn't be able to speak or read/write now, I remembered how tidy the language is compared to french.

The first hint was the group of letters `hdnd` that I had never seen before, but the confirmation came with `ausgehend` which is just a prefix-swapped version of `eingehend`. I verified with Google translate, it turns out I was correct.

**Pull request merged**